.semaphore/semaphore.yml

```diff
@@ -0,0 +1,64 @@
+version: v1.0
+name: Kelp test and build pipeline
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+
+blocks:
+  - name: "Test and build Kelp"
+    task:
+      prologue:
+        commands:
+          # Golang setup, required since we're building with Go versions < 1.11:
+          - export "SEMAPHORE_GIT_DIR=$(go env GOPATH)/src/github.com/stellar/${SEMAPHORE_PROJECT_NAME}"
+          - export "PATH=$(go env GOPATH)/bin:${PATH}"
+          - mkdir -vp "${SEMAPHORE_GIT_DIR}" "$(go env GOPATH)/bin"
+
+          # Checking out the repo:
+          - checkout
+
+          # Installing glide and deps:
+          - curl https://glide.sh/get | sh
+          - glide install
+
+          # Installing node.js 10.5:
+          - nvm install 10.5 && nvm use 10.5
+
+          # Running CCXT:
+          - docker run -p 3000:3000 -d franzsee/ccxt-rest:v0.0.4
+      jobs:
+        # Run the test suite + try building Kelp with Go 1.10
+        - name: Run package tests (1.10)
```

Comment:
I'm guessing that there's no way to avoid this repetition for testing on multiple versions of golang?
Do you know if we can have templates or something to avoid this repetition?